### PR TITLE
update wordCount cache in scheduled scraping jobs

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay  @raymondng  @martinraison 

update cache when:
1) successfully scraped article (new article or article changed)  ==> actual word count
2) unscrapable ==> -1
3) scrape error  ==> -1

do not upgrade if article is not changed.

### CODE REVIEW

File: kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala [hunk 1]
```diff
@@ -190,6 +209,9 @@ class ScrapeWorker @Inject() (
       helper.syncSaveNormalizedUri(latestUri.withState(NormalizedURIStates.SCRAPE_FAILED))
     }
     log.warn(s"[processURI] Error($httpStatus, $msg); errorURI=(${errorURI.id}, ${errorURI.state}, ${errorURI.url})")
+
+    updateWordCountCache(errorURI.id.get, None)
```
Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:213]: Say we had a good scrape previously and got a wordcount. Now we re-scrape and then we err out ... the count goes to 0, right?

Comment (yingjieMiao) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:213]: yes. goto -1 (undefined), to be exact.  Does it make sense to maintain old count?

Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:213]: As long as this does not show up in product, it's okay.

**Comment by raymondng** [comment]
lgtm

**Pull request merged**